packages/product-feed-luigis-box/src/Model/FeedItem/LuigisBoxFeedItem.php

```diff
@@ -99,81 +107,121 @@ public function getDescription(): ?string
     /**
      * @return string
      */
-    public function getLink(): string
+    public function getUrl(): string
     {
         return $this->url;
     }
 
     /**
      * @return string|null
      */
-    public function getImageLink(): ?string
+    public function getImageLinkS(): ?string
     {
-        return $this->imgUrl;
+        if ($this->imgUrl === null) {
+            return null;
+        }
+
+        return ImageUrlWithSizeHelper::limitWidthInImageUrl($this->imgUrl, static::SMALL_IMAGE_SIZE, static::SMALL_IMAGE_SIZE);
     }
 
     /**
-     * @return string
+     * @return string|null
      */
-    public function getAvailability(): string
+    public function getImageLinkM(): ?string
     {
-        return $this->availability;
+        if ($this->imgUrl === null) {
+            return null;
+        }
+
+        return ImageUrlWithSizeHelper::limitWidthInImageUrl($this->imgUrl, static::MEDIUM_IMAGE_SIZE, static::MEDIUM_IMAGE_SIZE);
     }
 
     /**
-     * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
+     * @return string|null
      */
-    public function getPrice(): Price
+    public function getImageLinkL(): ?string
     {
-        return $this->price;
+        if ($this->imgUrl === null) {
+            return null;
+        }
+
+        return ImageUrlWithSizeHelper::limitWidthInImageUrl($this->imgUrl, static::LARGE_IMAGE_SIZE, static::LARGE_IMAGE_SIZE);
+    }
+
+    /**
+     * @return string
+     */
+    public function getAvailabilityRankText(): string
+    {
+        return $this->availabilityText;
     }
 
     /**
      * @return int
      */
-    public function getCategoryId(): int
+    public function getAvailabilityRank(): int
```

Comment:
there are quite a lot of magic numbers in this method, could you please extract them into constants?

Reply:
btw, it is funny that `availableInDays` now can be only `0` or `null` :slightly_smiling_face: 
(based on `ProductAvailabilityFacade::getProductAvailabilityDaysByDomainId` implementation)

Reply:
I have added some constants. This functionality is very improvised as we have nothing of such kind implemented.

Reply:
ok, thanks for the fix and info :slightly_smiling_face: